ZEN-27916 Excessive logging in opentsdb

ISPHOST opened the pull request.

tgjonestx commented on services/Zenoss.core.full/Infrastructure/opentsdb/reader/-CONFIGS-/opt/opentsdb/src/logback.xml:
```diff
@@ -0,0 +1,74 @@
+    <appender-ref ref="QUERY_LOG"/>
+  </logger>
+  <logger name="org.apache.zookeeper" level="INFO"/>
+  <logger name="org.hbase.async" level="WARN"/>
```
Please add a comment here with the Jira ticket number to explain that setting `org.hbase.async` and `com.stumbleupon.async` to something higher than WARN can result in excessive logging under some conditions.

tgjonestx replied:
Process check - since ZEN-27916 is against `develop`and ZEN-26681 is against `support/5.3.x`, then either the branch for this PR should have been named `feature/ZEN-26681` or this PR should have been made against the `support/5.3.x` branch. Not a big deal this time, but something to keep in mind for future PRs

ISPHOST commented:
Changed log level settings for `org.hbase.async` and `com.stumbleupon.async` from INFO to WARN by default. On some environments, log level INFO is excessive logging under some conditions. Added possibility to change log level in CC. https://jira.zenoss.com/browse/ZEN-26681
